src/definitions/modules/dropdown.js

```diff
@@ -2712,11 +2721,14 @@ $.fn.dropdown = function(parameters) {
         },
 
         add: {
-          label: function(value, text, shouldAnimate) {
+          // Accepts a wrapper to preserve mutations made by onBeforeAdd callback
+          label: function(valueWrapper, shouldAnimate) {
```

Comment:
I think this is a breking change.
I recommend to **overloading** this function like below:

```js
var value, text;
if ($.isPlainObject(valueWrapper)) {
  value = valueWraper.value;
  text = valueWrapper.text;
} else {
  value = arguments[0];
  text = arguments[0];
  shouldAnimate = arguments[0];
}
```

Reply:
@vibridi Are you still interested to finish this PR?

Reply:
@lubber-de I'm sorry, I'm afraid I can't put in the hours at this time. I'm too busy with work/family. Feel free to close or take over. Thanks!

Reply:
Hey! I want to take this up! But I haven't dug into the code yet. I have very little idea about the project's structure and architectural concept (if any). What should I know before going through the source code? or can I dive right into it?